development/build/index.js

```diff
@@ -82,34 +82,38 @@ async function defineAndRunBuildTasks() {
     // scuttle on production/tests environment only
     const shouldScuttle = ['dist', 'prod', 'test'].includes(entryTask);
 
-    console.log(
-      `Building lavamoat runtime file`,
-      `(scuttling is ${shouldScuttle ? 'on' : 'off'})`,
-    );
-
-    // build lavamoat runtime file
-    await lavapack.buildRuntime({
-      scuttleGlobalThis: applyLavaMoat && shouldScuttle,
-      scuttleGlobalThisExceptions: [
-        // globals used by different mm deps outside of lm compartment
-        'toString',
-        'getComputedStyle',
-        'addEventListener',
-        'removeEventListener',
-        'ShadowRoot',
-        'HTMLElement',
-        'Element',
-        'pageXOffset',
-        'pageYOffset',
-        'visualViewport',
-        'Reflect',
-        'Set',
-        'Object',
-        'navigator',
-        'harden',
-        'console',
-        'Image', // Used by browser to generate notifications
-        // globals chrome driver needs to function (test env)
+    let scuttleGlobalThisExceptions = [
+      // globals used by different mm deps outside of lm compartment
+      'toString',
+      'getComputedStyle',
+      'addEventListener',
+      'removeEventListener',
+      'ShadowRoot',
+      'HTMLElement',
+      'Element',
+      'pageXOffset',
+      'pageYOffset',
+      'visualViewport',
+      'Reflect',
+      'Set',
+      'Object',
+      'navigator',
+      'harden',
+      'console',
+      'Image', // Used by browser to generate notifications
+      // globals sentry needs to function
+      '__SENTRY__',
+      'appState',
+      'extra',
+      'stateHooks',
+      'sentryHooks',
+      'sentry',
+    ];
+
+    if (entryTask === 'test') {
```

Comment:
There are two entry tasks for test builds: "test" and "testDev".

We also have constants for each of them. I have another PR open now where I've replaced the other entry task references in this file with constants: https://github.com/MetaMask/metamask-extension/pull/19506

Reply:
Ah, okay. I was going off the `shouldScuttle` array above. Your changes make sense. I'll wait until that PR is merged and then rebase this.